Microsoft.HealthVault.Fhir.UnitTests/ToHealthVaultTests/ObservationToHealthVaultWeightTests.cs

```diff
@@ -36,6 +36,21 @@ public void WeightToHealthVault_Successful()
         }
 
         [TestMethod]
+        public void HeightToHealthVault_Successful()
```

Comment:
The class is ObservationToHealthVaultWeightTests and in it HeightTests ? .. better to have a new class for heighttests ?